ddtrace/internal/telemetry/__init__.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from ...internal.utils.formats import asbool
+from ...internal.utils.formats import get_env
+from .data.integration import Integration
+from .data.payload import AppStartedPayload
+from .data.telemetry_request import TelemetryRequest
+from .data.telemetry_request import create_telemetry_request
+from .telemetry_writer import TelemetryWriter
+
+
+_TELEMETRY_WRITER = None  # type: Optional[TelemetryWriter]
+
+TELEMETRY_ENABLED = asbool(get_env("instrumentation_telemetry", "enabled", default=False))  # type: bool
```

Comment:
instead of `get_env` you can just do `asbool(os.environ.get("DD_INSTRUMENTATION_TELEMETRY_ENABLED", default=False))`

we are trying to deprecate `get_env`